### DISCUSSION TITLE
Data filtering

### DISCUSSION THREAD

**Discussion by pejota66** [Nesvilab/FragPipe-Analyst-doc#13] [Q&A] [answered]
Hello 

Does the Fragpipe-analyst performs the data filtering (i.e. removing proteins classified as Only identified by site,
Contaminants,  and Reverse)?

Is it possible to retrieve the R-script of the analyst and use it in R

Thank you

**Comment by hsiaoyi0504** [comment, marked as answer]
@pejota66 The R code of FragPipe-Analyst is here: https://github.com/MonashProteomics/FragPipe-Analyst.
For the data filtering part, contaminants and reverse peptides are removed but not necessary in FragPipe-Analyst. The majority of them were removed when you ran the FragPipe.
